hugr-py/src/hugr/std/__init__.py

```diff
@@ -1 +1,15 @@
 """Types and operations from the standard extension set."""
+
+from pathlib import Path
+
+from hugr.ext import Extension
+from hugr.serialization.extension import Extension as PdExtension
+
+_EXT_DIR = Path(__file__).parent / "_json_defs"
```

Comment:
Use `pkgutil`
https://stackoverflow.com/questions/6028000/how-to-read-a-static-file-from-inside-a-python-package/58941536#58941536